src/main/res/menu/log_list.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:bwq="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/switch_logs"
+        android:title="@string/log_android_title"
+        bwq:showAsAction="always" />
+
+      <item
+        android:id="@+id/menu_share"
+        android:title="@string/share_title"
+        bwq:actionProviderClass="android.support.v7.widget.ShareActionProvider"
+        bwq:showAsAction="always" />
```

Comment:
What does `bwq` stand for? I've just used `app` in the other menus.

Also, what does the action provider do exactly?